webpack.config.js

```diff
@@ -107,6 +107,9 @@ module.exports = {
       { test: /\.js$/, exclude: /node_modules/, loader: "babel-loader" },
     ],
   },
+  node: {
+    fs: "empty",
+  },
```

Comment:
I'm not 100% on the reason this is needed, but without it the project won't build with this error:
```
Error: Can't resolve 'fs' in '/Users/trevorcroxson/duwamish/node_modules/jsonlint-lines/lib'
```
The issue seems to be that we're bundling `mapbox-gl-draw` ourselves on this branch rather than importing the prebuilt version.

There's a long discussion about this issue here: https://github.com/webpack-contrib/css-loader/issues/447

The consensus seems to be to use the `fs: "empty"` trick.